inst/include/general_demographic_projection.hpp

```diff
@@ -102,12 +125,14 @@ void run_fertility_and_infant_migration(int time_step,
     state_next.base.p_total_pop(0, g) =
         births_sex * demog.survival_probability(0, g, time_step);
 
+    if (pars.base.options.proj_period_int == internal::PROJPERIOD_MIDYEAR) {
     // Assume 2/3 survival_probability rate since mortality in first six months higher
     // than second 6 months (Spectrum manual, section 6.2.7.4)
-    real_type migration_rate_a0 = demog.net_migration(0, g, time_step) *
-                                  (1.0 + 2.0 * demog.survival_probability(0, g, time_step)) /
-                                  3.0 / state_next.base.p_total_pop(0, g);
-    state_next.base.p_total_pop(0, g) *= 1.0 + migration_rate_a0;
+      real_type migration_rate_a0 = demog.net_migration(0, g, time_step) *
+                                    (1.0 + 2.0 * demog.survival_probability(0, g, time_step)) /
+                                    3.0 / state_next.base.p_total_pop(0, g);
```

Comment:
I think that more explanation / a clearer way of presenting this may be useful. I looked at the spectrum manual and I think that changing the note to read as  "// Assume 2/3 survival_probability rate since mortality in first six months of life for children < 1 is higher  // than second 6 months of life (Spectrum manual, section 6.2.7.4)"

For 131-133: 
I'm reading this as:
mig * (1/3 + 2/3 * surv) / pop

Is survival_probability actually a mortality probability? What does the 1/3 represent? Why doesn't this have to be averaged with the year before if it is a mid year calculation?

Reply:
Add migration_rate_a0 the intermediate data object rather than define here.

Reply:
Thanks very much, I have elaborated the comment and split up the calculation over two lines.

Surv is the survival probability and the way you have read is correct. In the net migration for all ages above one, half of the mortality is applied `migrant_survival = (0.5 + 0.5 * surv)`. But for infants, the assumption is they will spend more time in the high mortality of the first six months, so 2/3 of the annual mortality is applied.

Reply:
Sounds good, thanks!